include/temoto_context_manager/env_model_repository.h

```diff
@@ -87,6 +87,12 @@ class Item : public std::enable_shared_from_this<Item>
   {
     return children_;
   }
+
+  std::vector<std::shared_ptr<Item>>* getChildrenNonConst() 
```

Comment:
why not return reference instead of a pointer?
`std::vector<std::shared_ptr<Item>>& getChildrenNonConst()
{
  return children_;
}`